_includes/_markdown/install-os-lib.md

```diff
@@ -0,0 +1,38 @@
+{% altDetails require-info-box 'Getting OS-Lib' %}
+
+{% tabs oslib-install class=tabs-build-tool %}
+{% tab 'Scala CLI' %}
+You can require the entire toolkit in a single line:
+```scala
+//> using toolkit "latest"
+```
+
+Alternatively, you can require just a specific version of OS-Lib:
+```scala
+//> using dep "com.lihaoyi::os-lib:0.9.1"
```

Comment:
Why don't we propose adding dependencies to toolkit instead, everywhere? That's the way for Toolkit with other buildtools, at least for now.

Reply:
That's what I was doing ^^